bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | rev | cut -d',' -f1 | rev )
```

Comment:
huh, I read `man rev` as reversing the order of lines in a file, not reversing each line's contents, my mistake @fnothaft!

@heuermh's trick above seems maybe a little cleaner, but I'm agnostic at this point and have held up this PR long enough so whatever you want to do sgtm @fnothaft!